exercises/accumulate/accumulate_test.go

```diff
@@ -38,9 +38,7 @@ func TestAccumulate(t *testing.T) {
 	for _, test := range tests {
 		actual := Accumulate(test.given, test.converter)
 		if fmt.Sprintf("%s", actual) != fmt.Sprintf("%s", test.expected) {
```

Comment:
I might say we have to use `%q` here too, otherwise for the first test (expects an empty list) someone could pass by emitting a list of many empty strings (not that I *expect* anyone to do that, but for completeness...?)

Or can switch to reflect.DeepEqual I suppose?